src/main/java/org/terasology/metalrenegades/ai/event/CitizenDeathEvent.java

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.metalrenegades.ai.event;
+
+import org.terasology.entitySystem.event.Event;
+
+public class CitizenDeathEvent implements Event {
```

Comment:
This event is imported in `CitizenDeathSystem.java`, but is never actually fired or received anywhere. Please use this event or remove it altogether (probably keep it, it may be useful later on)